src/main/java/unicash/commons/util/AppUtil.java

```diff
@@ -10,10 +10,6 @@
  */
 public class AppUtil {
 
-    private AppUtil() {
-
-    }
-
```

Comment:
FYI, this is needed cos there will be a codecov path that checks for the constructor initialization but since these files all contain static methods, we use the private constructor to pass this path

Reply:
I see, I will add a comment there to prevent accidental deletion in the future

Reply:
Sounds good, thanks!